### PR TITLE
Add support for Abyssus Elite (D.Va Edition)

### PR DESCRIPTION
**Pull request by nightsky30**
Add support for Abyssus Elite (D.Va Edition) for #900 

Everything has been tested including setting all effects, brightness, dpi, poll rate, and reading values.

### CODE REVIEW

**Review by z3ntu — CHANGES_REQUESTED** [review]
most comments from your other PR #1068 also apply here :)

**Comment by nightsky30** [comment]
@z3ntu The same changes have been incorporated into this PR, and retested.  This one should be good to go as well :+1:

File: daemon/openrazer_daemon/hardware/mouse.py [hunk 1]
```diff
@@ -42,8 +45,7 @@ def _suspend_device(self):
 
         Get the current brightness level, store it for later and then set the brightness to 0
         """
-        self.suspend_args.clear()
-        self.suspend_args['brightness'] = (_da_get_logo_brightness(self))
+        self.brightness = (_da_get_logo_brightness(self))
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/mouse.py]: same comment as in the other PR ^^

**Comment by z3ntu** [comment]
CI output:
> [!] driver/razermouse_driver.c does not respect the agreed coding style.
> You can run ./scripts/format_source.sh to automatically format those files.

File: daemon/openrazer_daemon/hardware/mouse.py (hunk 1 above)
Comment (nightsky30) [reply daemon/openrazer_daemon/hardware/mouse.py]: Sorry about this.  Should be good now.

**Review by z3ntu — APPROVED** [review]
LGTM

**Pull request merged**